css/settings-jekyll.scss

```diff
@@ -0,0 +1,33 @@
+---
+---
+
+/*
+  As jekyll doesn't provide a way to include liquid templating constructs
+  in scss partials in the `_sass` folder, this file will serve for that
+  purpose. If there's ever the need to include a jekyll variable on a
+  stylesheet, it'll be done on this file. It should be kept organized
+  with the same sections/components that are on the `_sass` folder.
+*/
+
+@import "../node_modules/bootstrap/scss/variables";
+@import "../node_modules/bootstrap/scss/mixins";
+
+/**
+ * HOME Lead section
```

Comment:
Below this section name,could you also put the name of the file where the class is defined? 
like

```
/**
 * Specific services sections
 * _sass/_components--service.scss
 */
```